tests/standalone/urllib3Using.py

```diff
@@ -0,0 +1,34 @@
+import urllib3
+import json
+
+http = urllib3.PoolManager()
+r = http.request('GET', 'http://httpbin.org/robots.txt')
+def test_data():
```

Comment:
Mere defining functions as `test_*` does not execute them. The Nuitka test runner just runs things as a program, and compares outputs. Please note that another PR is covering this with enhancements, e.g. avoiding to use network, but localhost instead.

Reply:
Thanks very much for your reviews. I'll work on this and make amends.

Quick question:

How then will this be compatible with pytest if the function name doesn't begin with the word `test`. I'm asking this because, I tried making the package test to be compatible not only with Nuitka but with pytests as well as you suggested here: https://github.com/Nuitka/Nuitka/issues/229#issuecomment-460060109?